motion/autopilot/cfg/Autopilot.cfg

```diff
@@ -7,15 +7,15 @@ gen = ParameterGenerator()
 gains = gen.add_group("Gains")
 
 #          Name     	Type             Level Description   Default  Min   Max
-gains.add("PID_p", 	    double_t, 0,    "Proportional gain", 0.5,     0.5,  5.0)
-gains.add("PID_i", 	    double_t, 0,    "Integral gain", 	 0.1,     0.0,  5.0)
-gains.add("PID_d", 	    double_t, 0,    "Derivative gain", 	 0.0,     0.0,  3.0)
+gains.add("PID_p", 	    double_t, 0,    "Proportional gain", 0.5,     0.5,  50.0)
```

Comment:
If no max/min value is provided, the sliders in the rqt_reconfigure become basically useless. The old code had defined max limits, but since I do not know what the parameters will be for the new AUV, I left it at a max which should give enough overhead. The max value can always be changed to something realistic later on :smile: